test/libdevcore/CommonData.cpp

```diff
@@ -28,16 +28,72 @@
 using namespace std;
 using namespace dev::solidity;
 
+// TODO: Fix Boost...
+BOOST_TEST_DONT_PRINT_LOG_VALUE(dev::bytes);
```

Comment:
Maybe define this helper and use it instead?
```
#define CHECK_EQUAL_RANGE(x,y) { auto const& lhs = x; auto const& rhs = y; BOOST_CHECK_EQUAL_COLLECTIONS(lhs.begin(), lhs.end(), rhs.begin(), rhs.end()); }
```

Reply:
The error message output in that case is not really helpful.

Reply:
But the current hack is also fine I guess...

Reply:
Well could just use `BOOST_CHECK` and manually print also..